content/projects/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Apache Camel Projects
+---
+
+## Apache Camel K
+
+Apache Camel K is a lightweight integration framework built from Apache Camel that runs natively on Kubernetes and is specifically designed for serverless and microservice architectures. Users of Camel K can instantly run integration code written in Camel DSL on their preferred cloud (Kubernetes or OpenShift).
+
+For more information checkout the [Camel K manual](../../camel-k/latest/) and join the community on on the [Camel Users mailing list](../../community/mailing-list/) and have a look at the [Camel K GitHub repository](https://github.com/apache/camel-k/).
```

Comment:
We have turned on relative URLs in Hugo configuration here:

https://github.com/apache/camel-website/blob/21dbd4d74b5f0270a697e8de300eeff30dca14dd/config.toml#L4

That means that we can use absolute URLs in markdown, and they should be converted to relative URLs when rendered to HTML.

These links point to `../../` which for `/projects` where this `_index.md` is rendered to, would end up resolving to `/projects/../../`, which would be outside of the root.

```suggestion
For more information checkout the [Camel K manual](/camel-k/latest/) and join the community on on the [Camel Users mailing list](/community/mailing-list/) and have a look at the [Camel K GitHub repository](https://github.com/apache/camel-k/).
```